JAVA: compile fails, missing target

ucx master @ 9afd8c146682e98585d46f200b2f36a3e75dfba9

(happens in out-of-source-tree build)

```sh
$ make
...
Making all in test/examples
make[2]: Entering directory '/home/tc/src/ucx/ucx-tony/build/test/examples'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/tc/src/ucx/ucx-tony/build/test/examples'
Making all in bindings/java/src/main/native
make[2]: Entering directory '/home/tc/src/ucx/ucx-tony/build/bindings/java/src/main/native'
make[2]: *** No rule to make target 'org_ucx_jucx_ucp_UcpWorker.h', needed by 'all'.  Stop.
make[2]: Leaving directory '/home/tc/src/ucx/ucx-tony/build/bindings/java/src/main/native'
make[1]: *** [Makefile:728: all-recursive] Error 1
make[1]: Leaving directory '/home/tc/src/ucx/ucx-tony/build'
make: *** [Makefile:596: all] Error 2
$ cd build/bindings/java/src/main/native
$ make -k V=1
make: *** No rule to make target 'org_ucx_jucx_ucp_UcpWorker.h', needed by 'all'.
make: *** No rule to make target 'org_ucx_jucx_ucs_UcsConstants.h', needed by 'all'.
make: Target 'all' not remade because of errors.
```

--- a/bindings/java/src/main/native/Makefile.am
+++ b/bindings/java/src/main/native/Makefile.am
@@ -17,6 +17,9 @@ DISTCLEANFILES = org_ucx_jucx_ucp_UcpConstants.h \
                  org_ucx_jucx_ucp_UcpWorker.h \
                  org_ucx_jucx_ucs_UcsConstants.h
 
+org_ucx_jucx_ucp_UcpConstants.h:
+org_ucx_jucx_ucp_UcpWorker.h:
+org_ucx_jucx_ucs_UcsConstants.h:
 org_ucx_jucx_ucp_UcpContext.h:
 	$(MVNCMD) compile native:javah
 

